internal/abci/e2e_test.go

```diff
@@ -223,10 +223,10 @@ func TestCreateADI(t *testing.T) {
 	require.Equal(t, types.String("acc://RoadRunner"), r.ChainUrl)
 	require.Equal(t, types.Bytes(keyHash[:]), r.KeyData)
 
-	kg := n.GetSigSpecGroup("RoadRunner/foo-book")
-	require.Len(t, kg.SigSpecs, 1)
+	kg := n.GetKeyBook("RoadRunner/foo-book")
+	require.Len(t, kg.Pages, 1)
```

Comment:
:shrug: Would you prefer "KeyPages". It's a property of the `KeyBook` struct, so I thought it could be omitted, but I have no problem with making it "KeyPages".